infra/gcp/clusters/k8s-infra-prow-build/prow-build/main.tf

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright 2020 The Kubernetes Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+This file defines:
+- GCP Project k8s-infra-prow-build to hold a prow build cluster
+- GCP Service Account for k8s-infra-prow-build pods (bound via workload identity to a KSA of the same name)
+- GCP Service Account for boskos-janitor (bound via workload identity to a KSA of the same name)
+- GKE cluster configuration for prow-build
+- GKE nodepool configuration for prow-build
+*/
+
+locals {
+  project_id              = "k8s-infra-prow-build"
+  cluster_name            = "prow-build"           // The name of the cluster defined in this file
```

Comment:
Historically the prow team hasn't ever had more than one build cluster per GCP project.

Reply:
We have here a project named prow-build and a cluster named prow-build.  I guess I am skeptical that we'll never want to use some GKE change that requires rebuilding the cluster, is all.  If you're confident, I'll go with it, since it's likely you who has to deal with the mess if you are wrong :)